### DISCUSSION TITLE
How to Shrinke .wasm file size

### DISCUSSION THREAD

**Discussion by livstyle** [slint-ui/slint#4528] [General]
How to Shrinke .wasm file size or split .wasm to more than one files, one .wasm
file's size is so large;

**Comment by tronical** [comment]
This is a good question. It's not entirely evident to me, either. So I've converted this into a discussion - and if ideas come out of it then we can perhaps file issues to track.

In principle I think splitting the big .wasm file into multiple isn't straight-forward to do - in practice it's not supported out of the box by the Rust tooling. I think emscripten supports something like that as part of their "dlopen", where I _think_ both wasm modules share the same memory then. But I'm not sure this solves much to begin with.

One thing that would be interesting would be to maybe externalise the embedded font.
